docs: add info for `verifyGetters` flag

cvetanov opened the pull request:
Hi @havardh,

I struggled for quite a few hours to have `lombok` and `javaflow` work together, only to discover that the `verifyGetters` flag was the one causing my troubles (no getters in the classes since lombok adds them at compile time), so I updated the readme to have more info on this flag.

In case someone else has this issue, disabling the flag with `false` or omitting it completely worked fine.

Regards,
Goce

Commit:
docs: add info for `verifyGetters` flag

havardh commented:
@cvetanov Great. :)

So in principal I propose we keep the descriptive documentation in the `javaflow` repo and let this repo document the existence of properties and syntax for configuring in the maven pom. My reason becomes clear when you want to change the implementation of the flag, I think it is cumbersome to be required to update documentation in a separate repository. 

So what about adding a `flags` section to the javaflow Readme.md, which we can refer to from here ?

Furthermore, you said you struggled with this for hours (sorry about that :/). Is it something in the code, like error messages, we could improve that would cut the time spent on figuring this out?

/cc @misino

cvetanov commented:
Hi,

Regarding my struggles, they are mostly my fault because I just copied the example documentation without knowledge of the `verifyGetters` flag. The error message was descriptive in itself. 😄 

And regarding adding a `flags` section to the javaflow Readme, I am not really sure I understood you. As far as I checked, the `verifyGetters` flag is part of the [plugin](https://github.com/havardh/javaflow-maven-plugin/blob/master/src/main/java/com/github/havardh/javaflow/plugins/JavaflowMojo.java#L154-L156), not the javaflow engine, so it made sense to me to document it here. But, then again, I added implementation details in the documentation, so I might have been wrong there.

Are you sure about adding a `flags` section to javaflow? The `verifyGetters` property in the plugin has no connection to any flags or properties inside javaflow (except having the same name as the `--verifyGetters` argument in the CLI), it is used only in the plugin to add the `ClassGetterNamingVerifier`. In my opinion, it makes more sense to have the docs I added here and have a `verifiers` section in `javaflow` which explains the verifiers.

Regards,
Goce

havardh commented:
Yes, you are absolutely right in that my proposed `flags` section is wrong in this case. :)

So lets go with your proposal of adding a `verifiers` section in `javaflow` which can include the details on the `ClassGetterNamingVerifier`. And I guess this PR can be changed to describe that the `verifyGetters` tag toggles this verifier on/off and defaults to false and refer to the `javaflow` Readme for the details.

Do you agree?

cvetanov commented:
Agreed. Will do that in the following days.

Commit:
docs: update info for `verifyGetters` flag
